executor/batch_point_get.go

```diff
@@ -202,6 +202,7 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 				continue
 			}
 
+			e.memTracker.Consume(types.EstimatedMemUsage(idxVals, 1))
```

Comment:
I think this line can be put at the beginning of the loop.

First, `datumsContainNull(idxVals)` will be true if ANY datum in `idxVals` is Null.
Second, a Null datum object still consuming memory.